internal/pkg/cli/dialog/create_test_template.go

```diff
@@ -63,23 +62,29 @@ func (d *createTmplTestDialog) askTestName() (string, error) {
 	v, _ := d.Dialogs.Ask(&prompt.Question{
 		Label:       "Test Name",
 		Description: "Please enter a test name.",
-		Validator:   validateTestName,
+		Validator: func(val interface{}) error {
+			str := strings.TrimSpace(val.(string))
+			if len(str) == 0 {
+				return fmt.Errorf(`test name is required`)
+			}
+
+			if !regexpcache.MustCompile(template.IdRegexp).MatchString(str) {
+				return fmt.Errorf(`invalid name "%s", please use only a-z, A-Z, 0-9, "-" characters`, str)
+			}
+
+			if _, err := d.template.Test(str); err == nil {
+				return fmt.Errorf(`test "%s" already exists`, str)
+			} else {
+				if !strings.Contains(err.Error(), "not found in template") {
+					return err
+				}
+			}
```

Comment:
Ještě teda mi chyběla kontrola jestli test s daným jménem už neexistuje. 

![CleanShot 2022-09-09 at 16 34 16](https://user-images.githubusercontent.com/215660/189374818-478cb9a2-49e8-4e59-a1cd-4c2545bef44d.jpg)

Reply:
A ještě teda aby se to kontrolovalo i v neinteraktivním režimu: https://github.com/keboola/keboola-as-code/pull/783/commits/83043b82390883c6ec0ea6d6cbddc5071e26246d